consumergroup/offset_manager.go

```diff
@@ -201,7 +203,13 @@ func (zom *zookeeperOffsetManager) offsetCommitter() {
 			close(zom.closed)
 			return
 		case <-tickerChan:
-			zom.commitOffsets()
+			if err := zom.commitOffsets(); err != nil {
+				zom.cg.errors <- &sarama.ConsumerError{
+					Topic:     "",
+					Partition: -1,
+					Err:       err,
+				}
+			}
```

Comment:
`chan error` makes more sense, but that means making a breaking change to the API.  If you're cool with it, I'm happy to change it.